app/main/posts/views/post-view-list.directive.js

```diff
@@ -242,4 +258,40 @@ function PostListController(
         // Clear selected posts
         $scope.selectedPosts.splice(0);
     }
+
+    function getNewPosts() {
+        var existingFilters = PostFilters.getQueryParams($scope.filters);
+        var filterDate = moment(existingFilters.date_before).format('MMM Do YY');
+        var now = moment().format('MMM Do YY');
+        if (filterDate >= now) {
+            var mostRecentPostDate = $scope.recentPosts[0] ? $scope.recentPosts[0].post_date : $scope.posts[0].post_date;
+            existingFilters.date_after = mostRecentPostDate;
+            var query = existingFilters;
+            var postQuery = _.extend({}, query, {
```

Comment:
@crcommons please use $scope.filters.orderby and $scope.filters.order since those are the ones applied by the user.